src/dos/dos_programs.cpp

```diff
@@ -18,1656 +18,26 @@
 
 #include "programs.h"
 
-#include <cctype>
-#include <cstdlib>
-#include <cstring>
-#include <memory>
-#include <string>
-#include <vector>
-
-#include "bios_disk.h"
-#include "bios.h"
-#include "callback.h"
-#include "cdrom.h"
-#include "control.h"
-#include "cross.h"
-#include "dma.h"
-#include "dos_system.h"
-#include "drives.h"
-#include "fs_utils.h"
-#include "inout.h"
-#include "mapper.h"
-#include "mem.h"
 #include "program_autotype.h"
+#include "program_boot.h"
 #include "program_choice.h"
 #include "program_help.h"
+#include "program_imgmount.h"
+#include "program_intro.h"
+#include "program_keyb.h"
+#include "program_loadfix.h"
+#include "program_loadrom.h"
 #include "program_ls.h"
-#include "regs.h"
-#include "setup.h"
-#include "shell.h"
-#include "support.h"
-#include "string_utils.h"
-#include "../ints/int10.h"
-
```

Comment:
This is a very nice cleanup of includes - and applying them on an as-needed basis per program. 
:+1: